src/main/webapp/scripts/session-script.js

```diff
@@ -1,46 +1,82 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+import { Session } from "./session";
 
 /**
- * function openSessionInfo() displays the div container
- * that has information about the session.
+ * An array of who is currently in the session.
+ * @type {Array}
  */
-function openSessionInfo() {
-  document.getElementById('session-info-div').style.display = 'block'; 
-}
+let currentAttendees = [];
+
+/**
+ * Represents the current session, a Session object.
+ * @type {Session}
+ */
+let session;
+
+/**
+ * Represents (in miliseconds) how long the message that alerts users
+ * of any membership changes in the session is displayed. 
+ * @type {number}
```

Comment:
Period implies something piece of work is being done that is repetitive. From this description, it sounds like this dictates how long a user sees something. Theres nothing in your description that implies that work is being repeatedtly done. I guess, im still confused on what this number is for. What is the exact scenario? Is it dictating how often javascript will check for new attendees? Is it dictating how *long* a user sees a particular message?

Reply:
Yes the last point, how long a user sees a particular message. I guess it’s just finding the right word that doesn’t convey anything that’s periodic, because it isn’t meant to be.

Reply:
duration maybe?

Reply:
DISPLAY_DURATION_MS SGTM. Maybe change the word display to something a bit more descriptive